spec/models/question_answer_comment_spec.rb

```diff
@@ -86,7 +86,7 @@
     it 'supports emoji markdown engine' do
       qa_comment = Fabricate(:qa_comment, post: post, raw: ':grin:')
 
-      expect(qa_comment.cooked).to eq("<p><img src=\"/images/emoji/twitter/grin.png?v=#{Emoji::EMOJI_VERSION}\" title=\":grin:\" class=\"emoji only-emoji\" alt=\":grin:\"></p>")
+      expect(qa_comment.cooked).to eq("<p><img src=\"/images/emoji/twitter/grin.png?v=12\" title=\":grin:\" class=\"emoji only-emoji\" alt=\":grin:\" loading=\"lazy\" width=\"20\" height=\"20\"></p>")
```

Comment:
This now hardcodes the emoji version. Should it? Do we want it to break if core bumps the version?

Reply:
Oops good catch. I was copying and pasting some stuff from another branch and broke this.